R/PackageFunctionReporter.R

```diff
@@ -151,8 +151,12 @@ FunctionReporter <- R6::R6Class(
             if (is.null(self$pkg_name)) {
                 log_fatal('Must set_package() before extracting nodes.')
             }
-            nodes <- data.table::data.table(
-                node = as.character(unlist(utils::lsf.str(asNamespace(self$pkg_name))))
+            nodes <- data.table::data.table(node = as.character(
+                unlist(
+                    utils::lsf.str(pos = asNamespace(self$pkg_name)
+                                   )
```

Comment:
you can bring this `)` up to the same line as the `utils::lsf.str` for readability

Reply:
also can you please add an `importFrom utils lsf.str` to the roxy?